File: portofolio personal/assets/projects/ketupat.py
#Menggambar ketupat menggunakan python

sisi = int(input("Masukan angka : "))

count = 1
spasi = int(sisi/2)
count1 = count*sisi
spasi1 = 1
gambar = "$"

while True:
        if count < sisi :
                if count%2 :
                        print(" "*spasi, gambar*count)
                        count += 1
                        spasi -= 1
                else :
                        count += 1
        elif count1%2:
                print(" "*spasi1, gambar*count1)
                count1 -= 1
                spasi1 += 1
        else :
                count1 -= 1
        if count1 < 1 :
                break

print("hasil akhir")

File: portofolio personal/assets/projects/ketupat.cpp
#include <iostream>
using namespace std;

int main(){
	int m;
	
	cout << "menggambar mengunakan c++\n";
	cout << "masukan pola \n" ;
	cin >> m;
		cout << endl;
		cout << endl;
	
	for(int i = 1; i <= m; i++){
		for(int k = m; k >= i; k--){
			cout << " ";
		}
		for(int j = 1; j < i; j += 1){
			if(j==1){
				cout << "♡";
			}else if(j>=1){
				cout << "♡";
				cout << "♡";
			}
		}
		cout << endl;
		//cout << endl;
		//cout << endl;
		//cout << " ";
	}
	//cout << " ";
	for(int i = 1; i <= m; i++){
		for(int k = 1; k <= i; k++){
			cout << " ";
		}
		for(int j = m; j >= i; j -= 1){
			if(j>=m){
				cout << "♡";
			}else if(j>=1){
				cout << "♡";
				cout << "♡";
			}
		}
		cout << endl;
		//cout << i;
	}
	cin.get();
}